app/views/admin/index.html.erb

```diff
@@ -86,6 +86,7 @@
       <li><%= link_to 'Resources', admin_resource_files_path %></li>
       <li><%= link_to 'Site Settings', admin_settings_path %></li>
       <li><%= link_to 'Team members', admin_team_members_path %></li>
+      <li><%= link_to 'Testimonials', admin_testimonials_path %></li>
```

Comment:
Can do this later, but I was thinking of moving all the "Other" links that are about managing content like this under the "Content Management" section thats in master.

Reply:
Can do this now, no probs

Reply:
Have added them to the content management section, but in alphabetical order, which does intertwine them with the 3 new CMS ones. Let me know if you want to order them with pages / categories / sections grouped together @ldodds